### PR TITLE
#030: Fixing the footer

### PR DESCRIPTION
**Pull request by kareraolivier**
background have been changed to the right one
button text color is changed
accordion paddding is added
[#fix footer is done]

# PR Description

This PR Contain changes on background have been changed to the right one, button text color is changed, accordion padding is added 

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Changed the background
- [x] changed padding
- [x] Changed button color

## Description of tasks that were expected to be completed and that are not yet completed

- [x] None

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies
2. run `npm run build`
3. run `npm run start`
4. Open liveserver to run project

## Screenshots (If appropriate/applicable)
Screenshot Capture - 2023-08-24 - 15-16-22

Screenshot Capture - 2023-08-24 - 15-17-00

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: src/sections/footer.html [hunk 1]
```diff
@@ -16,17 +15,15 @@
                 <p class="text-white md:block hidden">stetig wachsender Markt und Talentpool</p>
```
Comment (devfabien) [inline src/sections/footer.html]: You didin't add the correct font family

File: src/sections/footer.html [hunk 2]
```diff
@@ -16,17 +15,15 @@
...
                         fill="white" fill-opacity="0.7" />
                 </svg>
                 <p class="text-white md:flex hidden">sehr gutes Investitionsklima</p>
```
Comment (devfabien) [inline src/sections/footer.html]: And also here add the correct font family

File: src/sections/footer.html [hunk 3]
```diff
@@ -52,20 +49,25 @@ <h1 class="lg:w-80 font-bold text-2xl">
...
+
+                <div class="md:py-16 px-4 py-4 w-full md:w-3/4 mx-auto md:grid md:grid-cols-2 justify-between gap-16 font-para"
+                    id="accordion4">
                 </div>
-                
+
                 <div class="py-3">
                     <div class="lg:w-1/2">
-                        <h1 class="font-bold md:text-2xl text-xl py-3">Nehmen Sie Kontakt zu uns auf!
+                        <h1 class="font-bold md:text-2xl text-xl py-3 md:py-6">Nehmen Sie Kontakt zu uns auf!
```
Comment (devfabien) [inline src/sections/footer.html]: Also here add the correct font family

File: src/sections/footer.html [hunk 4]
```diff
@@ -52,20 +49,25 @@ <h1 class="lg:w-80 font-bold text-2xl">
...
                         </h1>
                         <button type="button"
-                            class="bg-yellow-500 hover:bg-yellow-600 py-2 px-6 rounded-3xl shadow-sm font-semibold border-none flex gap-3 items-center"> <span><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6">
-                                <path d="M1.5 8.67v8.58a3 3 0 003 3h15a3 3 0 003-3V8.67l-8.928 5.493a3 3 0 01-3.144 0L1.5 8.67z" />
-                                <path d="M22.5 6.908V6.75a3 3 0 00-3-3h-15a3 3 0 00-3 3v.158l9.714 5.978a1.5 1.5 0 001.572 0L22.5 6.908z" />
-                              </svg>
-                              </span> Nachricht schreiben</button>
+                            class="text-backgroundColor bg-yellow-500 hover:bg-yellow-600 py-2 px-6 rounded-3xl shadow-sm font-semibold border-none flex gap-2 md:gap-6 items-center">
```
Comment (devfabien) [inline src/sections/footer.html]: Change the background color of the mail button

**Review by devfabien — CHANGES_REQUESTED** [review]
Most of changes are there but try to add those new styles i pointed out

File: src/sections/footer.html (hunk 1 above)
Comment (kareraolivier) [reply src/sections/footer.html]: Changed

File: src/sections/footer.html (hunk 2 above)
Comment (kareraolivier) [reply src/sections/footer.html]: done

File: src/sections/footer.html (hunk 4 above)
Comment (kareraolivier) [reply src/sections/footer.html]: done

File: src/sections/footer.html (hunk 3 above)
Comment (kareraolivier) [reply src/sections/footer.html]: i added them

**Review by devfabien — APPROVED** [review]
Looking good

**Review by kunda4 — APPROVED** [review]
Good job

**Pull request merged**